bootstrap/bootstrapper.ts

```diff
@@ -0,0 +1,70 @@
+import * as cluster from 'cluster';
+import * as os from 'os';
+import * as restify from 'restify';
+import Bugsnag from '@bugsnag/js';
+import { Controllers, Services } from './';
+import { Container } from 'inversify';
+import { InversifyRestifyServer } from 'inversify-restify-utils';
+import { PassportMiddleWare } from '../app/middleware/index';
+import { createConnection } from 'typeorm';
+import { env } from '../app/helpers';
+
+export class Bootstrapper {
+    public static bootstrap(): void {
+        if (cluster.isMaster) {
+            const numCpus = os.cpus().length;
+            for (let i = 0; i < numCpus; i++) {
+                cluster.fork();
+            }
+            cluster.on('exit', (): void => {
+                cluster.fork();
+            });
+            return;
+        }
+        this.bootstrapWorker();
+    }
+
+    private static bootstrapWorker(): void {
+        new Promise(
+            async (resolve, reject): Promise<void> => {
+                try {
+                    Bugsnag.start({
+                        apiKey: 'ef38e6e76834f9ebb8a0bfbc30621b3f',
```

Comment:
why is this hard-coded here?

Reply:
api is removed from code & key is re-generated